src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    private BetDao betDao;
+    @Inject
+    private UserDao userDao;
```

Comment:
let's receive from the console some user info as well, create a new User and add it to the storage as well